ops/docker/op-stack-go/Dockerfile

```diff
@@ -136,14 +142,23 @@ FROM --platform=$TARGETPLATFORM $TARGET_BASE_IMAGE AS op-node-target
 COPY --from=op-node-builder /app/op-node/bin/op-node /usr/local/bin/
 CMD ["op-node"]
 
-FROM --platform=$TARGETPLATFORM $TARGET_BASE_IMAGE AS op-challenger-target
+# Copy in kona mimicking how proofs-tools does.
```

Comment:
nit: We'll likely be able to remove proofs-tools once this is settled in as vm-runner can just use the op-challenger image. So maybe just:
```suggestion
# Make the kona docker image published by upstream available as a source to copy kona and asterisc from.
```